modules/scripts-utils/ace-adapter/src/main/resources/xml/xproc/library.xpl

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:library xmlns:p="http://www.w3.org/ns/xproc" 
+    xmlns:px="http://www.daisy.org/ns/pipeline/xproc" 
+    xmlns:pxi="http://www.daisy.org/ns/pipeline/xproc/internal" version="1.0"
+    xmlns:c="http://www.w3.org/ns/xproc-step">
+
+
+    <p:declare-step type="px:ace" name="main">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h1 px:role="name">ACE report generation</h1>
+            <p px:role="desc">Generate an accessibility check report using the Accessibility Checker for EPUB tool.</p>
+        </p:documentation>
+        <!-- anyFileURI to the epub -->
+        <p:option name="epub" required="true">
+            <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+                <h1 px:role="name">EPUB URI</h1>
+                <p px:role="desc">URI of the EPUB to control</p>
```

Comment:
"control" is not a good translation of "contrôler" in this context. "check" is better.